pallets/funding/src/mock.rs

```diff
@@ -414,13 +415,14 @@ construct_runtime!(
 // Build genesis storage according to the mock runtime.
 pub fn new_test_ext() -> sp_io::TestExternalities {
 	let mut t = frame_system::GenesisConfig::<TestRuntime>::default().build_storage().unwrap();
-
+	let ed = <TestRuntime as pallet_balances::Config>::ExistentialDeposit::get();
 	RuntimeGenesisConfig {
 		balances: BalancesConfig {
-			balances: vec![(
-				<TestRuntime as Config>::PalletId::get().into_account_truncating(),
-				<TestRuntime as pallet_balances::Config>::ExistentialDeposit::get(),
-			)],
+			balances: vec![
+				(<TestRuntime as Config>::PalletId::get().into_account_truncating(), ed),
+				(<TestRuntime as Config>::ContributionTreasury::get(), ed),
```

Comment:
Curiosity: why should the Contribution Treasury have at least ED?

Reply:
Good catch!
I assumed we could just use the PLMC ED to use a provider for a new pallet-assets account for that token, but we would run into the same problem we had before with regular participants.

So someone needs to pay for the account deposit by calling "touch". Should the issuer pay for it or the PLMC treasury?

Reply:
Should this be a one-off thing or should the account have enough PLMC for each project?

Reply:
Every new asset you want to transfer/mint to an account with pallet-assets, requires either increasing the provider count of the account (which you are limited to), or paying for the storage of that asset.

System accounts behave the same as regular accounts, so they need an ED for creating the account, and they have the provider count increased by other pallets like pallet-assets.

This won't be a one-off thing, since each project will have a different asset in pallet-assets, so will need a new provider count, or a new payment for the storage creation. 

I will make the issuer pay for it